public/stylesheets/main.css

```diff
@@ -1075,11 +1068,11 @@ html {
   -webkit-tap-highlight-color: transparent; }
 
 body {
-  font-family: "Lato", "Arial", sans-serif;
+  font-family: "Open Sans", Lato, "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
@KatieMFritz We can even talk about specific changes in the code, like this font stuff. :smile: